src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -962,33 +962,40 @@ private ImmutableArray<BoundExpression> BindInterpolatedStringParts(BoundUnconve
 
                 SyntaxNode placeholderSyntax;
                 uint valSafeToEscapeScope;
+                bool isSuppressed;
 
                 switch (argumentIndex)
                 {
-                    case BoundInterpolatedStringArgumentPlaceholder.InstanceParameter:
+                    case BoundInterpolatedStringArgumentPlaceholder.InstanceParameter when receiver != null:
                         placeholderSyntax = unconvertedString.Syntax;
-                        valSafeToEscapeScope = receiverEscapeScope;
+                        valSafeToEscapeScope = calledMember.RequiresInstanceReceiver()
```

Comment:
>valSafeToEscapeScope [](http://example.com/codeflow?start=24&length=20)

Is this change related to the stated goal of the PR, i.e. support for nullable analysis? Is there a change in behavior here?

Reply:
> Is this change related to the stated goal of the PR, i.e. support for nullable analysis? Is there a change in behavior here?

Yes, it's related, though this itself shouldn't cause a behavior change. I needed to pass through additional context from the receiver for the correct syntax node to report nullability warnings, and whether the receiver has a null suppression on it. I decided to simplify the existing receiver data to just pass the receiver itself.